Server/JokesOnYou.Web.Api/Repositories/UserRepository.cs

```diff
@@ -27,6 +49,11 @@ public async Task<User> GetUserAsync(string id)
             return user;
```

Comment:
shouldnt return user if he is deleted

Reply:
ah this is a weird one in Github. there are some lines missing above here. You can expand to see the missing lines.
DeleteUserAsync does not return a user :P
But you made me look at Looking up users and found a null error ;)